acquia_lift_profiles/acquia_lift_profiles.module

```diff
@@ -147,27 +147,35 @@ function acquia_lift_profiles_page_build(&$page) {
   }
   $js_path = variable_get('acquia_lift_profiles_js_path', '');
   $account_name = variable_get('acquia_lift_profiles_account_name', '');
-  $udf_mappings = variable_get('acquia_lift_profiles_udf_mappings', array());
+  $capture_identity = variable_get('acquia_lift_profiles_capture_identity', FALSE);
+  $mappings = variable_get('acquia_lift_profiles_udf_mappings', array());
+  $acquia_lift_profiles_vocabulary_mappings = variable_get('acquia_lift_profiles_vocabulary_mappings', array());
+  if (!empty($acquia_lift_profiles_vocabulary_mappings)) {
+    $mappings['vocabulary'] = $acquia_lift_profiles_vocabulary_mappings;
```

Comment:
Why are these grouped one level deeper in the structure under 'vocabulary'? I don't see how this could work in the js. Can you add a qunit test for this? Also, it no longer makes sense to call these vocabulary mappings as they don't use taxonomy vocabularies anymore. Let's call them field mappings.